scenarios/kubernetes_bazel.py

```diff
@@ -84,7 +84,15 @@ def query(kind, selected_pkgs, changed_pkgs):
 
     selection = '//...'
     if selected_pkgs:
-        selection = 'set(%s)' % ' '.join(selected_pkgs)
+        # negated packages start with '-'
+        pkgs = []
+        negated_pkgs = []
+        for pkg in selected_pkgs:
+            if pkg.startswith('-'):
+                negated_pkgs.append(pkg)
+            else:
+                pkgs.append(pkg)
+        selection = 'set(%s) - set(%s)' % (' '.join(pkgs), ' '.join(negated_pkgs))
```

Comment:
this may not be correct, still.

I'm pretty sure bazel supports queries like
`bazel test -- //... -//foo/... //foo/bar/... -//foo/bar/baz/...`

which this logic doesn't currently handle correctly.

> Target patterns may be preceded by a single dash ('-'), in which case Bazel subtracts them from the set of targets accumulated by preceding patterns. Note that this means order matters.